tests/utils.js

```diff
@@ -466,6 +474,22 @@ const prepServices = async (config) => {
   await runAndLogApiStartupMessage('User contact doc setup', setUserContactDoc);
 };
 
+//non browser tests
+const prepServicesNative = async () => {
```

Comment:
I think we shouldn't just add all this duplicated code, and simply refactor the code that already exists. 
Also "native" isn't an appropriate suffix here, it doesn't communicate what the difference is between this function and `prepServices`.